games/w.yaml

```diff
@@ -444,3 +444,22 @@
   - 'http://wxlasertank.sourceforge.net/screenshots/main.png'
   - 'http://wxlasertank.sourceforge.net/screenshots/game.png'
   - 'http://wxlasertank.sourceforge.net/screenshots/game_petes.png'
+  
+  
+-name: Walic
```

Comment:
there are YAML syntax errors here; for lists YAML requires a dash and a space, see the other entries for examples
https://docs.ansible.com/ansible/latest/reference_appendices/YAMLSyntax.html#yaml-basics